server/event.go

```diff
@@ -138,6 +139,7 @@ type splitOperator struct {
 	Origin *metapb.Region `json:"origin"`
 	Left   *metapb.Region `json:"left"`
 	Right  *metapb.Region `json:"right"`
+	End    time.Time      `json:"end"`
```

Comment:
this operator just records split, the TiKV report split mean already finished split, use end time in here.